src/components/Modals/FiatRamps/hooks/useFiatRampCurrencyList.ts

```diff
@@ -1,4 +1,5 @@
 import { avalancheAssetId } from '@shapeshiftoss/caip'
+import { uniqBy } from 'lodash'
```

Comment:
```suggestion
import uniqBy from 'lodash/uniqBy'
```

at some stage we'll lint rule this for tree shaky reasons